common/production_methods/imperia_urban.txt

```diff
@@ -70,16 +70,16 @@ pm_trams = {
 	building_modifiers = {
 		workforce_scaled = {
 			goods_input_engines_add = 1
-			goods_input_coal_add = 5
-			goods_output_transportation_add = 15
+			goods_input_coal_add = 4
+			goods_output_transportation_add = 12
 			
-			#profit = (15*30)-((1*60)+(5*30)) = 240 = 12480
+			#profit = (12*30)-((1*60)+(4*30)) = 180*52 = 9360
 		}
 
 		level_scaled = {
-			building_employment_engineers_add = 100
-			building_employment_machinists_add = 400
-			building_employment_laborers_add = 1000
+			building_employment_engineers_add = 50
+			building_employment_machinists_add = 350
+			building_employment_laborers_add = 800
```

Comment:
I don't like that we're adding 400 jobs per urban center for this level. It is counterintuitive to the design of almost every other PM in the game to add jobs for improved output. We should be making its 1200 laborers for stagecoaches if this is the path we want to take.

Reply:
the rickshaw transport PM is very much a fluff PM that represents manpower-intensive transports like palanquins, litters and sedan chairs alongside rickshaws. i consequently tried to reflect that in its low input goods but higher manpower requirements.